nixnet/database/_dbc_signal_value_table.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import six
+import typing  # NOQA: F401
+
+from nixnet import _funcs
+from nixnet import constants
+
+
+class DbcSignalValueTable(collections.Mapping):
+    """Collection for accessing a DBC signal value table."""
+
+    def __init__(self, handle):
+        # type: (int) -> None
+        self._handle = handle
+
+    def __repr__(self):
+        return '{}(handle={})'.format(type(self).__name__, self._handle)
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self._handle == typing.cast(DbcSignalValueTable, other)._handle
+        else:
+            return NotImplemented
+
+    def __ne__(self, other):
+        result = self.__eq__(other)
+        if result is NotImplemented:
+            return result
+        else:
+            return not result
+
+    def __hash__(self):
+        return hash(self._handle)
+
+    def __len__(self):
+        return len(self._value_table)
+
+    def __iter__(self):
+        return self.keys()
+
+    def __getitem__(self, key):
+        # type: (typing.Text) -> int
+        """Return the value.
+
+            Args:
+                Value description.
+            Returns:
+                Value
+        """
+        if isinstance(key, six.string_types):
+            return self._value_table[key]
+        else:
+            raise TypeError(key)
+
+    def keys(self):
+        """Return all value descriptions in the collection.
+
+            Yields:
+                An iterator to all value descriptions in the collection.
+        """
```

Comment:
I wish there was a better name for keys than 'value descriptions'...

Reply:
Me too.